bundle/src/main/java/com/adobe/acs/commons/mcp/model/ArchivedProcessFailure.java

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright 2017 Adobe.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.adobe.acs.commons.mcp.model;
+
+import com.adobe.acs.commons.fam.Failure;
+import javax.inject.Inject;
+import org.apache.commons.lang3.exception.ExceptionUtils;
+import org.apache.sling.api.resource.Resource;
+import org.apache.sling.models.annotations.DefaultInjectionStrategy;
+import org.apache.sling.models.annotations.Model;
+
+/**
+ * Persisted failure information.
+ */
+@Model(adaptables = Resource.class, defaultInjectionStrategy = DefaultInjectionStrategy.OPTIONAL)
+public class ArchivedProcessFailure {
```

Comment:
Actually no -- but it is a model bean of sorts.  Should I make an IMPL sub-folder under model and hide it in there?